chore: Add privacy manifest to RN SDK readme. (#444)
launchdarkly/js-core: packages/sdk/react-native/README.md

yarn && yarn ios-go
```

## Privacy manifest

> Please use the [privacy manifest](https://github.com/launchdarkly/ios-client-sdk/blob/v9/LaunchDarkly/LaunchDarkly/PrivacyInfo.xcprivacy) included in the LaunchDarkly iOS Client SDK.

Reviewer: Is it possible to make this link version-independent? Otherwise you'll have to update it here every time iOS has a new major version, which may be difficult to remember.

Author: i used v9 because that looks like the `main` branch in the ios repo. I'm asking @keelerm84 in slack.

Reviewer 2: I think if you used `HEAD` it would always point to the most recent one, even if the default branch change. https://github.com/launchdarkly/ios-client-sdk/blob/HEAD/LaunchDarkly/LaunchDarkly/PrivacyInfo.xcprivacy

> Please use the [privacy manifest](https://github.com/launchdarkly/ios-client-sdk/blob/v9/LaunchDarkly/LaunchDarkly/PrivacyInfo.xcprivacy) included in the LaunchDarkly iOS Client SDK.
> 
> At WWDC23, Apple introduced the concept of Privacy Manifests. The privacy manifest included with the LaunchDarkly iOS Client SDK describes our data usage with respect to the minimum case of data collection. You will have to update your own privacy manifest if you choose to collect more data in your implementation than the minimum for our SDK to function.
> 

Reviewer: ```suggestion
At WWDC23, Apple introduced the concept of Privacy Manifests. The privacy manifest describes our data usage with respect to the minimum case of data collection. You will have to update your own privacy manifest if you choose to collect more data in your implementation than the minimum for our SDK to function.

For the LaunchDarkly React Native SDK, you can use the [privacy manifest](https://github.com/launchdarkly/ios-client-sdk/blob/v9/LaunchDarkly/LaunchDarkly/PrivacyInfo.xcprivacy) included in the LaunchDarkly iOS Client SDK. The data usage information is the same for the two SDKs.

```
Suggestion to flip these paragraphs -- describe what this is first (like we do in the iOS readme) and then point folks to the file to use.

> To learn more about Privacy Manifests, please refer to [Apple Developer Documention.](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests).

Reviewer: ```suggestion
To learn more about Privacy Manifests, please refer to [Apple Developer Documentation](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests).
```
spelling & punctuation nit

> To learn more about Privacy Manifests specifically in terms of react-native, please refer to this [discussion](https://github.com/react-native-community/discussions-and-proposals/discussions/776).

Reviewer: ```suggestion
To learn more about Privacy Manifests specifically for React Native, please refer to this [discussion](https://github.com/react-native-community/discussions-and-proposals/discussions/776).
```

> 

Author: I took this mostly from the iOS sdk. Appreciate any improvements/corrections, thank you.
